src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -163,8 +163,11 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             src="{{workItem.relationalData.creator.attributes.imageURL + '&s=20'}}"
             id="WI_details_reporter_img" />          
           <div class="pull-left marginR10">                          
-              <span id="WI_details_reporter_user"> 
-                {{workItem.relationalData?.creator?.attributes.fullName || 'No creator found'}} 
```

Comment:
So,
There could be another solution ,Not showing up "creator" when creator (does'nt exists ) WI is not yet created ? there is no point of showing up for every new WI creation for this !
Any input for this ? @debloper @nimishamukherjee